guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -108,8 +108,8 @@
     "        - labels.csv: 'B-Person', 'I-Person', 'I-Person', 'L-Person', 'O', 'O', 'O', 'U-Country'\n",
```

Comment:
<blockquote><code>Data preparation and model training workflows using arcgis.learn have a dependency on spaCy. This can be installed using conda as follows:&nbsp;conda install -c esri arcgis fastai spacy scikit-image</code></blockquote><p> </p><p>Suggestion: This pre-requisite can be removed as spacy is already installed as a dependency while installing the other packages.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>